acmwebsite/controllers/root.py

```diff
@@ -59,7 +59,23 @@ def index(self):
         meetings = DBSession.query(Meeting).filter(
             Meeting.date > datetime.datetime.now() - datetime.timedelta(hours=3)
         ).order_by(Meeting.date).limit(2)
-        return dict(page='index', meetings=meetings)
+        try:
```

Comment:
I don't like this. I think we should be doing an `if` on the count of the Banner query.